frontends/web/src/contexts/DarkmodeProvider.tsx

```diff
@@ -77,7 +77,10 @@ export const DarkModeProvider = ({ children }: TProps) => {
           const { darkmode, ...frontend } = config.frontend;
           setConfig({
             backend: config.backend,
-            frontend,
+            frontend: {
+              ...frontend,
+              darkmode: undefined,
+            },
```

Comment:
looking at this, I think `backend: config.backend,` isn't needed anymore as setConfig will take care of it..

Reply:
Yes!

Reply:
as already discussed DM, frontend and getConfig could be removed. but the detectDarkTheme backend api seems to return false even when my OS is dark (macOS). So I was hesitant to change more as this PR is just a small regression bug fix.